docs/UserGuide.md

```diff
@@ -172,18 +176,32 @@ Examples:
 *  `edit 1 d/Must include username, email and password fields by/13-12-2020 12:00`
 edits the description of the 1st deliverable to be `Must include username, email and password fields`
 and its deadline to be `13-12-2020 12:00`.
-*  `edit 2 c/2,4,5` edits the contacts of the 2nd deliverable to be `2,4,5`.
+*  `edit 2 c/Jordan Woods, Betsy Crowe, Jeremey` edits the contacts of the 2nd deliverable to be `Jordan Woods, Betsy Crowe, Jeremey`.
 
 #### Marking a deliverable as completed: `done`
 
-Marks the specified deliverable from your deliverable list as done.
+Marks the specified deliverable from your displayed deliverable list as done.
 
 Format: `done INDEX`
 * `INDEX` is the index number of the deliverable in your displayed deliverable list.
 * `INDEX` must be a positive integer.
 
 Example:
-* `done 1` marks the 1st deliverable in your deliverable list as done.
+* `done 1` marks the 1st deliverable in your displayed deliverable list as done.
+
+#### Viewing a deliverable: `view`
+
+Displays more details of the specified deliverable from your displayed deliverable list.
+
+Format: `view INDEX`
+* `INDEX` is the index number of the deliverable in your displayed deliverable list.
+* `INDEX` must be a positive integer.
+
+Example:
+* `view 2` views the 2nd deliverable in your displayed deliverable list.
```

Comment:
actually this is a bit weird, should it be displays or some other word

Reply:
yeah it is abit weird but im not sure what else to leave it as. Its not wrong tho